<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolicyCreator</class>
 <widget class="QMainWindow" name="PolicyCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fromZoneLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From Zone</string>
    </property>
   </widget>
   <widget class="QLabel" name="toZoneLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To Zone</string>
    </property>
   </widget>
   <widget class="QListView" name="fromZoneList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="toZoneList">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="policyNameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Policy Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="policyNameEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourceAddressEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sourceAddressLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourceAddressEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sourceAddressLabel_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination Address</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="thenCheckBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Permit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commitBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
